src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -142,7 +179,7 @@ const SelectPanel = props => {
             }}
           >
             {/* render default header as fallback */}
-            {slots.header || <SelectPanel.Header />}
+            {slots.header || <SelectPanelHeader />}
```

Comment:
I think of these as equivalent so really not meant as a suggestion to change, just is always fun to use new features when an opportunity comes up 😂 

```suggestion
            {slots.header ?? <SelectPanelHeader />}
```